docker/nuxeo-explorer-export-docker/skaffold.yaml

```diff
@@ -0,0 +1,32 @@
+# (C) Copyright 2019-2020 Nuxeo (http://nuxeo.com/) and others.
+#
+# Contributors:
+#     Kevin Leturc <kleturc@nuxeo.com>
+#     Anahide Tchertchian
+#
+apiVersion: skaffold/v1beta11
+kind: Config
+build:
+  tagPolicy:
+    envTemplate:
+      template: "{{.DOCKER_REGISTRY}}/nuxeo/{{.IMAGE_NAME}}:{{.VERSION}}"
+  artifacts:
+    - image: nuxeo-explorer-export
+      context: docker/nuxeo-explorer-export-docker
+      kaniko:
+        buildArgs:
+          BASE_IMAGE: ${DOCKER_REGISTRY}/nuxeo/nuxeo:${NUXEO_IMAGE_VERSION}
```

Comment:
`NUXEO_IMAGE_VERSION` is not referenced in the maven plugin configuration, isn't it an issue?

Reply:
currenlty skaffold is only used in the pipeline (where i fill this env variable explicitly): the idea is to pass this version as a param explicitly from some scripts (the export one or the release one), and retrieve it from the nuxeo-parent version for the docker build (see above comment to avoid using 11.x https://github.com/nuxeo/nuxeo-explorer/pull/33#discussion_r496617343)